Pipelines/src/cards/gaussian.jl

```diff
@@ -0,0 +1,158 @@
+- `_μ_1, _μ_2, ..., _μ_n`: Gaussian means.
+
+Returns:
+- A SimpleTable (Dict{String, AbstractVector}) with Gaussian parameters.
+"""
+function gaussian_train(g::GaussianCard)
+    μs = range(0, stop=1, length=g.means)  # Means normalized between 0 and 1
+    σ = round(step(μs) * g.coef, digits=4)
+    params = Dict("_σ" => [σ], "_d" => [g.max])
+    for (i, μ) in enumerate(μs)
+        params["_μ_$i"] = [μ]
+    end
+    return SimpleTable(params)
+end
+
+"""
+gaussian_train(repo::Repository, g::GaussianCard; schema=nothing) -> SimpleTable
+
+Generates a SimpleTable containing:
+- `_σ`: Standard deviation for the Gaussian transformations.
+- `_d`: Normalization value.
+- `_μ_1, _μ_2, ..., _μ_n`: Gaussian means.
+
+Args:
+- `repo`: Repository instance (necessary for compatibility but unused).
+- `g`: GaussianCard instance.
+- `source`: Source table name (necessary for compatibility but unused).
+- `schema`: Optional schema name (necessary for compatibility but unused).
+
+Returns:
+- A SimpleTable (Dict{String, AbstractVector}) with Gaussian parameters.
+"""
+train(repo::Repository, g::GaussianCard, source::AbstractString; schema = nothing) = gaussian_train(g::GaussianCard)
+
+"""
+evaluate(repo::Repository, g::GaussianCard, params_tbl::SimpleTable, (source, target)::Pair; schema=nothing)
+
+Applies the Gaussian transformation defined by the GaussianCard to the source table.
+
+Steps:
+1. Preprocesses the column using the specified method (e.g., `"dayofyear"`, `"hour"`, `"identity"`).
+2. Temporarily registers the Gaussian parameters (`params_tbl`) using `with_table`.
+3. Joins the source table (with preprocessing applied) with the params table via a CROSS JOIN.
+4. Computes Gaussian-transformed columns (`gaussian_1`, `gaussian_2`, ...).
+5. Selects only the required columns (original and transformed), excluding intermediate values.
+6. Replaces the target table with the final results.
+
+Args:
+- `repo`: Repository instance.
+- `g`: GaussianCard instance.
+- `params_tbl`: SimpleTable containing Gaussian parameters (`_σ`, `_d`, `_μ_#`).
+- `source`: Source table name.
+- `target`: Target table name for the transformed data.
+- `schema`: Optional schema name.
+"""
+function evaluate(
+    repo::Repository, g::GaussianCard, params_tbl::SimpleTable, (source, target)::Pair; schema = nothing
+)
+    preprocess = get(GAUSSIAN_METHODS, g.method, nothing)
+    if isnothing(preprocess)
+        throw(ArgumentError("Method $(g.method) is not supported for GaussianCard"))
+    end
+
+    converted = [
+        string(g.column, "_", g.suffix, "_", i) => gaussian_transform(
+            Get("_transformed"), Get(Symbol("_μ_$i")), Get(:_σ), Get(:_d)
+        ) for i in 1:g.means
+    ]
+    
+    with_table(repo, params_tbl; schema) do tbl_name
+        transform_query = With(
+            :transformed_table => From(source) |>
+                Define(preprocess(Get(g.column)) |> As("_transformed"))
+        )
+        join_query = From(:transformed_table) |>
```

Comment:
```suggestion
        join_query = From(source) |>
            Define("_transformed" => preprocess(Get(g.column))) |>
```
        
```

Reply:
maybe generated name instead of `_transformed` to avoid conflicts? eg `string(uuid4())`